tests/validate_must_gather_test.go

```diff
@@ -172,12 +172,6 @@ var _ = Describe("validate the must-gather output", func() {
 				"dmesg",
 				"ip.txt",
 				"lspci",
-				"nft-ip-filter",
```

Comment:
Worth mentioning maybe please that the nft files are not collected due to 
https://bugzilla.redhat.com/show_bug.cgi?id=2025750

will be fixed by https://github.com/kubevirt/must-gather/pull/116
(depends on this PR first)